msgs/action/PlanPickPlace.action

```diff
@@ -1,7 +1,23 @@
 # goal
+
+# planning group properties
+string arm_group_name
+string hand_group_name
+
+# robot model and links
+string eef_name
+string hand_frame
+
+# object and surfaces
 string object_id
-string end_effector
+string[] support_surfaces
+
+# picking
+string grasp_provider_plugin_name
```

Comment:
Same for this. As an end user, what do I have to do about this? What is the default?